test/project-setup-test.js

```diff
@@ -0,0 +1,20 @@
+// import mocha from 'mocha';
+// import chai from 'chai';
+
+const mocha = require ('mocha');
+const chai = require('chai');
+const expect = chai.expect;
+
+describe('mybrand api tasks',(done)=>{
+
+    it('starts the server listening to port 3000',()=>{
+
+       expect('app running on port 3000');
```

Comment:
Import the app and add a real `expact` that tests both the status of the response and the res object itself. You can even match the message being `returned`.